Pod/Classes/PagingMenuOptions.swift

```diff
@@ -20,6 +20,7 @@ public class PagingMenuOptions {
     public var menuPosition: MenuPosition = .Top
     public var menuHeight: CGFloat = 50
     public var menuItemMargin: CGFloat = 20
+    public var menuItemDividerImage: UIImage? = nil
```

Comment:
tiny thing, but could you not assign nil? Since its optional, `public var menuItemDriverImage: UIImage?` would be fine.